apps/system/test/unit/nfc_handover_manager_test.js

```diff
@@ -1,25 +1,27 @@
 'use strict';
 
-/* globals MocksHelper, MockBluetooth, MockBluetoothTransfer,
-           MockNavigatorSettings, NDEF, NfcConnectSystemDialog,
-           MockDOMRequest, MockL10n, NDEFUtils, BaseModule, MockMozNfc,
-           NfcUtils, MockNavigatormozSetMessageHandler, MockLazyLoader,
-           MockService */
-
+/* globals MocksHelper,
+           MockNavigatorSettings, NDEF,
+           MockL10n, NDEFUtils, BaseModule, MockMozNfc, NfcUtils,
+           MockNavigatormozSetMessageHandler, MockLazyLoader,
+           Service, MockBTAdapter, MockBluetooth, MockDOMRequest */
 require('/shared/test/unit/mocks/mock_navigator_moz_set_message_handler.js');
 require('/shared/test/unit/mocks/mock_moz_ndefrecord.js');
 require('/shared/test/unit/mocks/mock_moz_nfc.js');
 require('/shared/test/unit/mocks/mock_l10n.js');
 require('/shared/js/nfc_utils.js');
+require('/shared/test/unit/mocks/mock_event_target.js');
+require('/shared/test/unit/mocks/mock_dom_request.js');
 require('/shared/test/unit/mocks/mock_notification_helper.js');
 requireApp('system/shared/test/unit/mocks/mock_navigator_moz_settings.js');
 requireApp('system/test/unit/mock_system_nfc_connect_dialog.js');
 requireApp('system/shared/test/unit/mocks/mock_event_target.js');
 requireApp('system/shared/test/unit/mocks/mock_dom_request.js');
 requireApp('system/test/unit/mock_lazy_loader.js');
-requireApp('system/shared/test/unit/mocks/mock_service.js');
-requireApp('system/test/unit/mock_bluetooth_transfer.js');
-requireApp('system/test/unit/mock_bluetooth.js');
+// requireApp('system/shared/test/unit/mocks/mock_service.js');
+// requireApp('system/test/unit/mock_bluetooth_transfer.js');
+// requireApp('system/test/unit/mock_bluetooth.js');
```

Comment:
mock_bluetooth.js is used for mock bluetooth.js , not mozBluetooth, so it's a missuse here